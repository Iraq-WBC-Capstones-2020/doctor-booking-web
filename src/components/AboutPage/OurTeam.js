import React from 'react';
import { Container, Row, Col } from 'react-bootstrap';
import TeamMember from './TeamMember';
function OurTeam() {
  return (
    <Container className="abSection">
      <Row>
        <Container className="text-center mt-5 section-title">
          <h1>Our Team</h1>
        </Container>
      </Row>
      <Row >
        <Col className="mt-5" lg="6">
          <TeamMember />
        </Col>
        <Col className="mt-5" lg="6">
          <TeamMember />
        </Col>
        <Col className="mt-5" lg="6">
          <TeamMember />
        </Col>
        <Col className="mt-5" lg="6">
          <TeamMember />
        </Col>
      </Row>
    </Container>
  );
}

export default OurTeam;
