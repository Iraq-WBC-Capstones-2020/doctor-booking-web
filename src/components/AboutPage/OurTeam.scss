@import '../../styles/variables';
.member-name {
  color: $primaryColor;
}
.section-title {
  color: $primaryColor;
  font-weight: bold;
}
.social-media {
  margin: 0 auto;
  img {
    width: 25px;
    fill: $primaryColor;
  }
  ul {
    width: 120px;
  }
}
