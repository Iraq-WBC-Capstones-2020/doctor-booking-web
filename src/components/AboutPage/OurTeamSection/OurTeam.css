.member-name {
  color: #026fc2;
}

.section-title {
  color: #026fc2;
  font-weight: bold;
}

.social-media {
  margin: 0 auto;
}

.social-media img {
  width: 25px;
  fill: #026fc2;
}

.social-media ul {
  width: 120px;
}
/*# sourceMappingURL=OurTeam.css.map */
