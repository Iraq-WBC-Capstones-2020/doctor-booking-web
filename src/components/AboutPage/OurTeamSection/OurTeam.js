import React from 'react';
import { Container, Row, Col } from 'react-bootstrap';
import TeamMember from './TeamMember';
import { useTranslation } from 'react-i18next';
function OurTeam() {
  const { t, i18n } = useTranslation(['translation', 'validation']);

  return (
    <Container className="abSection ourTeam">
      <Row>
        <Container className="text-center mt-5 section-title">
          <h1>{t('validation:aboutPage.ourTeam.headLine')}</h1>
        </Container>
      </Row>
      <Row>
        <Col className="mt-5" lg="6">
          <TeamMember />
        </Col>
        <Col className="mt-5" lg="6">
          <TeamMember />
        </Col>
        <Col className="mt-5" lg="6">
          <TeamMember />
        </Col>
        <Col className="mt-5" lg="6">
          <TeamMember />
        </Col>
      </Row>
    </Container>
  );
}

export default OurTeam;
