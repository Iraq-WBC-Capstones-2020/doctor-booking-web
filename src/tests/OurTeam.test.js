import React from 'react';
import { render } from '@testing-library/react';
import OurTeam from '../components/AboutPage/OurTeamSection/OurTeam';
import renderer from 'react-test-renderer';
test('our team section is rendering ', () => {
  render(<OurTeam />);
});

test('our team section renders correctly', () => {
  const tree = renderer.create(<OurTeam />).toJSON();
  expect(tree).toMatchSnapshot();
});
