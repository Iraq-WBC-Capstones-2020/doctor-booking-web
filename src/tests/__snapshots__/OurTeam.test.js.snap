// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`our team section renders correctly 1`] = `
<div
  className="mb-5 container"
>
  <div
    className="row"
  >
    <div
      className="text-center mt-5 section-title container"
    >
      <h1>
        Our Team
      </h1>
    </div>
  </div>
  <div
    className="mt-5 row"
  >
    <div
      className="mt-5 col-lg-6"
    >
      <div
        className="row"
      >
        <div
          className="d-flex align-items-center flex-column col-6"
        >
          <div>
            <img
              alt="team member"
              className="img-fluid"
              src="avatar.svg"
            />
          </div>
          <div
            className="social-media mt-2"
          >
            <ul
              className="list-unstyled d-flex justify-content-between"
            >
              <li
                className="d-inline-block"
              >
                <a
                  href="#"
                >
                  <img
                    alt="github icon"
                    src="github.svg"
                  />
                </a>
              </li>
              <li
                className="d-inline-block"
              >
                <a
                  href="#"
                >
                  <img
                    alt="facebook icon"
                    src="facebook.svg"
                  />
                </a>
              </li>
              <li
                className="d-inline-block"
              >
                <a
                  href="#"
                >
                  <img
                    alt="instagram icon"
                    src="instagram.svg"
                  />
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          className="member-info d-flex flex-column col-6"
        >
          <h2
            className="member-name"
          >
            John Doe
          </h2>
          <p
            className="member-role"
          >
            Member Role
          </p>
          <p
            className="member-description"
          >
            some description here about the person and here is more text just for test
          </p>
        </div>
      </div>
    </div>
    <div
      className="mt-5 col-lg-6"
    >
      <div
        className="row"
      >
        <div
          className="d-flex align-items-center flex-column col-6"
        >
          <div>
            <img
              alt="team member"
              className="img-fluid"
              src="avatar.svg"
            />
          </div>
          <div
            className="social-media mt-2"
          >
            <ul
              className="list-unstyled d-flex justify-content-between"
            >
              <li
                className="d-inline-block"
              >
                <a
                  href="#"
                >
                  <img
                    alt="github icon"
                    src="github.svg"
                  />
                </a>
              </li>
              <li
                className="d-inline-block"
              >
                <a
                  href="#"
                >
                  <img
                    alt="facebook icon"
                    src="facebook.svg"
                  />
                </a>
              </li>
              <li
                className="d-inline-block"
              >
                <a
                  href="#"
                >
                  <img
                    alt="instagram icon"
                    src="instagram.svg"
                  />
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          className="member-info d-flex flex-column col-6"
        >
          <h2
            className="member-name"
          >
            John Doe
          </h2>
          <p
            className="member-role"
          >
            Member Role
          </p>
          <p
            className="member-description"
          >
            some description here about the person and here is more text just for test
          </p>
        </div>
      </div>
    </div>
    <div
      className="mt-5 col-lg-6"
    >
      <div
        className="row"
      >
        <div
          className="d-flex align-items-center flex-column col-6"
        >
          <div>
            <img
              alt="team member"
              className="img-fluid"
              src="avatar.svg"
            />
          </div>
          <div
            className="social-media mt-2"
          >
            <ul
              className="list-unstyled d-flex justify-content-between"
            >
              <li
                className="d-inline-block"
              >
                <a
                  href="#"
                >
                  <img
                    alt="github icon"
                    src="github.svg"
                  />
                </a>
              </li>
              <li
                className="d-inline-block"
              >
                <a
                  href="#"
                >
                  <img
                    alt="facebook icon"
                    src="facebook.svg"
                  />
                </a>
              </li>
              <li
                className="d-inline-block"
              >
                <a
                  href="#"
                >
                  <img
                    alt="instagram icon"
                    src="instagram.svg"
                  />
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          className="member-info d-flex flex-column col-6"
        >
          <h2
            className="member-name"
          >
            John Doe
          </h2>
          <p
            className="member-role"
          >
            Member Role
          </p>
          <p
            className="member-description"
          >
            some description here about the person and here is more text just for test
          </p>
        </div>
      </div>
    </div>
    <div
      className="mt-5 col-lg-6"
    >
      <div
        className="row"
      >
        <div
          className="d-flex align-items-center flex-column col-6"
        >
          <div>
            <img
              alt="team member"
              className="img-fluid"
              src="avatar.svg"
            />
          </div>
          <div
            className="social-media mt-2"
          >
            <ul
              className="list-unstyled d-flex justify-content-between"
            >
              <li
                className="d-inline-block"
              >
                <a
                  href="#"
                >
                  <img
                    alt="github icon"
                    src="github.svg"
                  />
                </a>
              </li>
              <li
                className="d-inline-block"
              >
                <a
                  href="#"
                >
                  <img
                    alt="facebook icon"
                    src="facebook.svg"
                  />
                </a>
              </li>
              <li
                className="d-inline-block"
              >
                <a
                  href="#"
                >
                  <img
                    alt="instagram icon"
                    src="instagram.svg"
                  />
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          className="member-info d-flex flex-column col-6"
        >
          <h2
            className="member-name"
          >
            John Doe
          </h2>
          <p
            className="member-role"
          >
            Member Role
          </p>
          <p
            className="member-description"
          >
            some description here about the person and here is more text just for test
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
